integration-tests/src/test/java/com/datastax/oss/driver/api/core/metadata/NodeMetadataIT.java

```diff
@@ -58,7 +56,50 @@ public void should_expose_node_metadata() {
     assertThat(node.getDistance()).isSameAs(NodeDistance.LOCAL);
     assertThat(node.getHostId()).isNotNull();
     assertThat(node.getSchemaVersion()).isNotNull();
+    assertThat(node.getLastResponseTimeNanos()).isEqualTo(-1);
```

Comment:
Would also be worth validating that `getUpSince` is not equal to -1.

Reply:
I also forced the node down and back up to check that the time is updated.